docs/source/models.rst

```diff
@@ -2943,26 +2977,34 @@ For `transformers` inference, there are two ways to pass in additional arguments
         generate_signature_output(model, data),
     )
 
-    # Define an inference_config
-    inference_config = {
+    # Define an model_config
+    model_config = {
```

Comment:
can we rename these references to ``load_config`` please?

Reply:
@BenWilson2 are you suggesting to name the concept `load_config` instead of `model_config`? I have mixed feelings about that. Specially consider the syntax when we say `mlflow.pyfunc.log_model(...., load_config=..)`. Indicating how to load in an instruction that saves things doesn't look good to me. Any particular reason `model_config` doesn't look appealing?

Reply:
`model_config` is fine